### DISCUSSION TITLE
Sensor Data

### DISCUSSION THREAD

**Discussion by gmcfalls** [br-g/openf1#65] [Ideas]
Tire Temperature, brake temperature, G force, fuel level, downforce.  It would be awesome to have feeds from other sensors.

**Comment by PrestonHager** [comment]
Unfortunately this isn't possible yet. The ingestor and API is describe in the [System Architecture](https://openf1.org/#system-architecture) page. All of the data is processed from the F1 livetiming site which currently only pushes the sensor data for RPM, speed, gear, DRS, and throttle and brake percentage. If F1 adds more public data, the ingestor is sure to process and store this data too.

**Comment by gmcfalls** [comment]
Thanks a lot for the response.  I marked this one as Closed.
